packages/collector/test/tracing/sdk/opt_in/test.js

```diff
@@ -0,0 +1,78 @@
+/*
+ * (c) Copyright IBM Corp. 2024
+ */
+
+'use strict';
+
+const expect = require('chai').expect;
+const path = require('path');
+
+const supportedVersion = require('@instana/core').tracing.supportedVersion;
+const config = require('../../../../../core/test/config');
+const ProcessControls = require('../../../test_util/ProcessControls');
+const globalAgent = require('../../../globalAgent');
+const { retry, delay } = require('../../../../../core/test/test_util');
+
+const mochaSuiteFn = supportedVersion(process.versions.node) ? describe : describe.skip;
+
+mochaSuiteFn('tracing/sdk/opt_in', function () {
```

Comment:
suggestions: 

- folder name "sdk/rootExitSpans"
- app names: app_default.js, app_with_entry.js
- I assume you wanted to proof that both cases still work?

Reply:
done